recipes-devtools/upm/upm_0.7.1.bb

```diff
@@ -16,16 +16,14 @@ inherit distutils-base pkgconfig python-dir cmake
 CFLAGS_append_edison = " -msse3 -mfpmath=sse"
 
 FILES_${PN}-doc += " ${datadir}/upm/examples/"
-RDEPENDS_${PN} += " mraa"
+RDEPENDS_${PN} += " mraa mraa-java"
 
 # override this in local.conf to get a subset of bindings.
 # BINDINGS_pn-upm="python"
 # will result in only the python bindings being built/packaged.
 
-#Disable the python package generation until we can get it building consistently on all platforms/architectures.
-# note, we can rely on BINDINGS_${PN}-mraa if we want these controlled by 1 variable
-#BINDINGS ?= "python nodejs java"
-BINDINGS ?= "nodejs java"
+# Build all language bindings by default
+BINDINGS ?= "python nodejs java"
```

Comment:
The python issue is random, so I'm not surprised it would build now with this tiny change.  I suspect the build issue will return sometime in the future though.